src/version.js

```diff
@@ -147,14 +149,16 @@ function nextReleaseVersion(defaultReleaseType, tagPrefix, fallbackToNoPrefixSea
   core.info(`Tag Prefix: '${tagPrefix}'`);
   core.info(`Next Release Version: ${nextReleaseVersion}`);
 
-  return nextReleaseVersion;
+  return [nextReleaseVersion, priorReleaseVersion];
```

Comment:
Now that we're adding another distinct value to the return of these `next*Version` functions, I think they deserve to be refactored so that they return an object with distinct properties.

The `nextReleaseVersion` value is already being disassembled into distinct parts by the code in `main.js` around line 86 and then reassembled in different ways.

Maybe these functions should return instances of the SemVer class (comes from the node-semver package `const SemVer = require('semver/classes/semver')`) in an object with two properties like the following:
```
nextVersion: {
    tagPrefix: string   // 'v', whatever is passed as the tag-prefix input, or ''
    semver: SemVer      // the SemVer of the next version
priorVersion: {
    tagPrefix: string
    semver: SemVer
}
```
The `SemVer` class has properties `raw`, `major`, `minor`, `patch`, and `prerelease` which is an array of pre-release identifiers (our prerelease version numbers would have two pre-release identifiers).

Then in `main.js` something similar to:
```
const outputs = {
      NEXT_VERSION: `${result.nextVersion.tagPrefix}${result.nextVersion.semver}`,
      NEXT_VERSION_NO_PREFIX: result.nextVersion.semver,
      NEXT_MINOR_VERSION: `${result.nextVersion.tagPrefix}${result.nextVersion.semver.major}.${result.nextVersion.semver.minor}`,
      NEXT_MINOR_VERSION_NO_PREFIX: `${result.nextVersion.semver.major}.${result.nextVersion.semver.minor}`,
      NEXT_MAJOR_VERSION: `${result.nextVersion.tagPrefix}${result.nextVersion.semver.major}`,
      NEXT_MAJOR_VERSION_NO_PREFIX: `${result.nextVersion.semver.major},

      NEXT_VERSION_SHA: sha,
      PRIOR_VERSION: `${result.priorVersion.tagPrefix}${result.priorVersion.semver}`

Reply:
Or if not the `SemVer` class itself, at least an object with properties like `major`, `minor`, `prereleaseIdentifiers`, `buildIdentifiers`, and a `toString()` that concatenates those all together with their appropriate `.`/`-`/`+` separators.

Reply:
Are you requesting for these refactoring changes to be made within this PR or a subsequent PR?

Reply:
In this PR.